Tasks/AzureRmWebAppDeploymentV3/task.json

```diff
@@ -17,7 +17,7 @@
     "version": {
         "Major": 3,
         "Minor": 4,
-        "Patch": 6
+        "Patch": 9
```

Comment:
Why are we skipping patch versions 7 and 8?

Reply:
cherrypicked from master. so had the same version.